bindings/rust/s2n-tls-sys/src/internal.rs

```diff
@@ -25,3 +25,9 @@ extern "C" {
         cert_chain_pem_size: u32,
     ) -> ::libc::c_int;
 }
+extern "C" {
+    pub fn s2n_config_new_minimal() -> *mut s2n_config;
+}
+extern "C" {
+    pub fn s2n_config_load_system_certs(config: *mut s2n_config) -> ::libc::c_int;
+}
```

Comment:
remind me. are we not stabilizing these ever or not now? Why not just stabilize them now?

Reply:
There's a separate PR to stabilize these APIs: https://github.com/aws/s2n-tls/pull/3985. I wanted to stabilize them in the same PR that updates the documentation with our recommended usage on s2n_config_new vs s2n_config_new_minimal.

Reply:
Yeah, I think it would have to remove the definitions in internal.rs. Whichever one gets merged first I'll have to rebase the other one on top of it. If it makes more sense to publish the APIs first I can wait to merge this one.